test/e2e/scale.go

```diff
@@ -183,6 +183,9 @@ func ScaleToWithin(t *testing.T, scale int, duration time.Duration, latencies La
 
 		case err := <-doneCh:
 			if err != nil {
+				// If we don't do this first, then we'll see tons of 503s from the ongoing probes
+				// as we tear down the things they are probing.
+				defer pm.Stop()
```

Comment:
Why not next to `pm := NewProberManager`?

Reply:
See some lines below this change, we want to explicitly Stop the probermanager and after successfully stopping it check the SLO.

Reply:
I see. And I don't see a pretty way out :)